features/bootstrap/HooksContext.php

```diff
@@ -7,229 +7,261 @@
-	 */
-	public function iExpectTheFollowingFiltersAdded( TableNode $table ) {
-		$filters  = $table->getHash();
-		$defaults = array(
-			'filter'    => '',
-			'callback'  => '',
-			'priority'  => 10,
-			'arguments' => 1,
-		);
-		foreach ( $filters as $filter ) {
-			$filter += $defaults;
-			WP_Mock::expectFilterAdded(
-				$filter['filter'],
-				$filter['callback'],
-				$filter['priority'],
-				$filter['arguments']
-			);
-		}
-	}
-
-	/**
-	 * @Given I expect the following filters not to be added:
-	 */
-	public function iExpectTheFollowingFiltersNotToBeAdded( TableNode $table ) {
-		foreach ( $this->getFiltersWithDefaults( $table ) as $filter ) {
-			WP_Mock::expectFilterNotAdded( $filter['filter'], $filter['callback'] );
-		}
-	}
-
-	/**
-	 * @When I add the following filters:
-	 */
-	public function iAddTheFollowingFilters( TableNode $table ) {
-		foreach ( $this->getFiltersWithDefaults( $table ) as $filter ) {
-			add_filter(
-				$filter['filter'],
-				$filter['callback'],
-				$filter['priority'],
-				$filter['arguments']
-			);
-		}
-	}
-
-	/**
-	 * @Given I expect filter :filter to respond to :thing with :response
-	 */
-	public function iExpectFilterToRespondToWith( $filter, $thing, $response ) {
-		WP_Mock::onFilter( $filter )->with( $thing )->reply( $response );
-	}
-
-	/**
-	 * @Given I expect filter :filter to respond with :response
-	 */
-	public function iExpectFilterToRespondWith( $filter, $response ) {
-		$this->iExpectFilterToRespondToWith( $filter, null, $response );
-	}
-
-	/**
-	 * @When I apply the filter :filter with :with
-	 */
-	public function iApplyFilterWith( $filter, $with ) {
-		$this->iApplyFilterWithData( $filter, new TableNode( array( array( $with ) ) ) );
-	}
-
-	/**
-	 * @When I apply the filter :filter with:
-	 */
-	public function iApplyFilterWithData( $filter, TableNode $table ) {
-		$row = $table->getRow( 0 );
-		array_unshift( $row, $filter );
-		$this->filterResults[ $filter ] = call_user_func_array( 'apply_filters', $row );
-	}
-
-	/**
-	 * @Then The filter :filter should return :value
-	 */
-	public function theFilterShouldReturn( $filter, $value ) {
-		\PHPUnit\Framework\Assert::assertArrayHasKey( $filter, $this->filterResults );
-		\PHPUnit\Framework\Assert::assertEquals( $this->filterResults[ $filter ], $value );
-	}
-
-	private function getActionsWithDefaults( TableNode $table ) {
-		$actions  = $table->getHash();
-		$defaults = array(
-			'action'    => '',
-			'callback'  => '',
-			'priority'  => 10,
-			'arguments' => 1,
-		);
-		foreach ( $actions as &$action ) {
-			$action += $defaults;
-		}
-		unset( $action );
-
-		return $actions;
-	}
-
-	private function getFiltersWithDefaults( TableNode $table ) {
-		$filters  = $table->getHash();
-		$defaults = array(
-			'filter'    => '',
-			'callback'  => '',
-			'priority'  => 10,
-			'arguments' => 1,
-		);
-		foreach ( $filters as &$filter ) {
-			$filter += $defaults;
-		}
-		unset( $filter );
-
-		return $filters;
-	}
```

Comment:
looks like this portion of the file was accidentally formatted perhaps by your editor- we plan to introduce some codestyle / phpcs automation to avoid style inconsistencies in the future - would you mind restoring the lines that have no code/logic change please?